src/github/pullRequestOverview.ts

```diff
@@ -464,20 +294,23 @@ export class PullRequestOverviewPanel {
 	}
 
 	private async addReviewers(message: IRequestMessage<void>): Promise<void> {
+		if (!issueAsPullRequest(this._item)) {
```

Comment:
can this check be removed? shouldn't `this._item` always be a PR?

Reply:
Leftover check from while refactoring was in progress. Removed!